flink-end-to-end-tests/flink-end-to-end-tests-common/pom.xml

```diff
@@ -65,6 +65,11 @@ under the License.
 			<version>${project.version}</version>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
```

Comment:
Could you please update the commit message?
I'd also splits the container modifications and the test environments into 2 commits.